src/components/MultiSelectProfessores.tsx

```
@@ -0,0 +1,87 @@
+import { useState } from "react"
+import { Command, CommandItem, CommandList, CommandInput } from "@/components/ui/command"
+import { Popover, PopoverContent, PopoverTrigger } from "@/components/ui/popover"
+import { Badge } from "@/components/ui/badge"
+import { X } from "lucide-react"
+
+const professores = [
+  { label: "João Silva", value: "joao" },
```

ESLint

[error] Replace `"João·Silva",·value:·"joao"` with `'João·Silva',·value:·'joao'`

(prettier/prettier)